Cloud Application Manager/Appliance/appliance-with-ec2-instance-role.md

```diff
@@ -0,0 +1,177 @@
+{{{
+"title": "CAM Appliance with EC2 Instance Role",
+"date": "03-08-2019",
+"author": "Roberto Santamaria",
+"keywords": ["aws", "iam", "appliance", "role"],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+- [Overview](#overview)
+- [Audience](#audience)
+- [Prerequisites](#prerequisites)
+- [Additional information](#additional-information)
+- [Create the Policy for the EC2 Instance Role](#create-the-policy-for-the-ec2-instance-role)
+- [Create the EC2 Instance Role](#create-the-ec2-instance-role)
+- [Attach the IAM Role to the Appliance EC2 Instance](#attach-the-iam-role-to-the-appliance-ec2-instance)
+- [Enable the EC2 Instace Role on CAM](#enable-the-ec2-instace-role-on-cam)
+- [Create a Policy and Role for the managed account](#create-a-policy-and-role-for-the-managed-account)
+- [Create a CAM Provider](#create-a-cam-provider)
+
+### Overview
+
+This article shows how to leverage an EC2 Instance Role as an authorization mechanism for all AWS API calls from a CAM Appliance. EC2 Instance Roles ease credential management and increase security for applications running in EC2 instances. You can learn more about them in [the AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html).
```

Comment:
Hey @robersantamaria, I think the comment inside the parentheses actually is good because later on we can just refer to the appliance and don't use the long name, don't you think?